docs/source/guide/pec-5-theory.myst

```diff
@@ -0,0 +1,156 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# What is the theory behind PEC?
+
+
+Probabilistic error cancellation (PEC) [[1-3]](#references) is a noise-aware error mitigation technique which is
```

Comment:
Here and elsewhere in the file: what about using the common References section at the end, with  `{cite}`ref1`` format?

Reply:
Applied here: f908f64